### ISSUE TITLE
How to reproduce the results in Table IX about 3DAES

### ISSUE THREAD

**Issue by AugustLHHHHH** [ShuGuoJ/3DAES#2]
Hi，I have read the paper and code。I understand that training the whole 3DAES involves three steps，the first step is to train the autoencoder，then using the parameters of encoder and random sample pair generation to train siamese network, finally, the rectified feature is classified by logistic regression。

I have completed the first two steps according to your instructions, So can you friendly share the code to reproduce Table IX about 3DAES? Because I need to compare OA 、AA and kappa with other methods. 

Appreciate your help.
Thanks and regards.

**Comment by ShuGuoJ** [comment]
Dear Mr. Liu:


Thanks for your attention to our work.


In the last step, all samples including both training and test samples are input into trained 3DAES, where they all are transformed into corresponding hidden features, which are used to train a simple logistical regression implemented in Sklearn Library. Then, classification results of test samples can be calculated by the trained classifier and all values (OA, AA, and kappa) can be get as well.


Shuguo Jiang.





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ShuGuoJ/3DAES"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月8日(星期二) 下午3:34
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[ShuGuoJ/3DAES] How to reproduce the results in Table IX about 3DAES (Issue #2)





 
Hi，I have read the paper and code。I understand that training the whole 3DAES involves three steps，the first step is to train the autoencoder，then using the parameters of encoder and random sample pair generation to train siamese network, finally, the rectified feature is classified by logistic regression。
 
I have completed the first two steps according to your instructions, So can you friendly share the code to reproduce Table IX about 3DAES? Because I need to compare OA 、AA and kappa with other methods.
 
Appreciate your help.
 Thanks and regards.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by lucas2606-rs** [comment]
when you finally classify with Logistic Regression, is the train and test set same as training siamese network?

**Comment by ShuGuoJ** [comment]
Yes, they keep consistent with those in training the siamese network.
